usr/src/cmd/zpool/zpool_vdev.c

```diff
@@ -549,7 +550,7 @@ get_replication(nvlist_t *nvroot, boolean_t fatal)
 	uint_t c, children;
 	nvlist_t *nv;
 	char *type;
-	replication_level_t lastrep, rep, *ret;
+	replication_level_t lastrep = {0}, rep, *ret;
```

Comment:
This should really be 3 different lines, for clarity.  But I suppose it's only slightly worse than the current code.